atlasdb-workload-server/src/test/java/com/palantir/atlasdb/workload/transaction/InMemoryValidationStoreTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload.transaction;
+
+import static com.palantir.atlasdb.workload.transaction.WorkloadTestHelpers.TABLE;
```

Comment:
Not sure if this goes against practice, but I noticed myself copying this to a lot of different files -- figured making a 'constants' class for this would help, but defer to the reviewer! If this is OK practice, will go update everything else to use this.